### DISCUSSION TITLE
Is a mobile app in the roadmap?

### DISCUSSION THREAD

**Discussion by nagarrido** [benjaminjonard/koillection#1188] [Ideas]
I'm not an app developer so I do not know how accurate is this, but wouldn't a mobile app be superior over the PWA if we used the phone camera? To take collection photos, or maybe to scan barcodes, etc

**Comment by PerpSearch** [comment]
I've been using it on my mobile and it's actually works pretty well for me, but the ability to use the phone cam to add images to a collection would be an excellent enhancement.

**Comment by benjaminjonard** [comment]
Hello, no mobile app is planned.

You should be able to access the phone camera from the PWA.
<img src="https://github.com/user-attachments/assets/34051bd0-e575-4816-9a35-cfd701a9effe"  width="200"  />

Which phone and OS version do you have?
I saw that Chrome on Android 14 might have a bug (?) where the camera is no longer accessible.
Edit: apparently this behavior was introduced in Chrome 116


As for scanning barcodes, I considered it some time ago but couldn't find anything sufficiently reliable to extract object data from the barcode.
